static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -327,3 +393,17 @@ const StyledLink = styled(Link)`
   ${LinkStyles}
   color: ${p => p.theme.gray300};
 `;
+
+const Flex = styled('div')`
```

Comment:
I would just drop this and include `display: flex` on stuff that's inheriting it